header.go

```diff
@@ -91,6 +92,11 @@ func readHeader(r *bufio.Reader, p *Part) (textproto.MIMEHeader, error) {
 	buf := &bytes.Buffer{}
 	tp := textproto.NewReader(r)
 	firstHeader := true
+	headerDeclarationRegex, err := regexp.Compile("^[a-zA-Z0-9\\-]+\\: ")
```

Comment:
This could also be a global variable `var headerDeclarationRegex = regex.MustCompile("^[a-zA-Z0-9\\-]+\\: ")`, saving some computation time. Drawback, should it ever fail it will cause a panic rather than an error that can be handled time by time